Sources/XCMetricsClient/Machine Name Reader/MachineNameReader.swift

```diff
@@ -37,17 +37,12 @@ class HashedMacOSMachineNameReader: MachineNameReader {
         if encrypted {
             return Host.current().localizedName?.md5()
         }
-        return removeLocalizedSingleQuotesAndSpaces(from: Host.current().localizedName)
+        return normalizing(Host.current().localizedName)
     }
 
-    func removeLocalizedSingleQuotesAndSpaces(from input: String) -> String {
-        // Define a set of localized variations of single quotes
-        let localizedSingleQuotes: Set<Character> = [
-            "‘", "’", "`", "ʹ", "′", "‛", "❛", "❜", "＇", "'"
-        ]
-
-        // Iterate through each character in the input string
-        return input.filter { !localizedSingleQuotes.contains($0) && !$0.isWhitespace }
+    private func normalizing(_ input: String) -> String {
+        let allowedChars = Set("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLKMNOPQRSTUVWXYZ1234567890")
```

Comment:
Nit - extra `K` in string
```suggestion
        let allowedChars = Set("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890")
```